src/net/gateway-archive.ts

```diff
@@ -0,0 +1,90 @@
+import { FileApi } from "../api/file";
+import { ContentUri } from "../wrappers/content-uri"
+import {
+    VOCDONI_MAINNET_ENTITY_ID,VOCDONI_RINKEBY_ENTITY_ID, VOCDONI_GOERLI_ENTITY_ID, VOCDONI_XDAI_ENTITY_ID, VOCDONI_SOKOL_ENTITY_ID, XDAI_ENS_REGISTRY_ADDRESS, XDAI_PROVIDER_URI, XDAI_CHAIN_ID,
+    SOKOL_CHAIN_ID, SOKOL_PROVIDER_URI, SOKOL_ENS_REGISTRY_ADDRESS, XDAI_STG_ENS_REGISTRY_ADDRESS, VOCDONI_XDAI_STG_ENTITY_ID
+} from "../constants"
+import { TextRecordKeys } from "../models/entity"
+import { IGateway } from "./gateway";
+import { DVoteGatewayResponseBody } from "./gateway-dvote";
+import { IGatewayPool } from "./gateway-pool";
+import { keccak256 } from "@ethersproject/keccak256"
+
+export type EthNetworkID = "mainnet" | "rinkeby" | "goerli" | "xdai" | "sokol"
```

Comment:
This is already defined in `src/common.ts` (on main), we can import it from there

Reply:
Ok, I was checking why I did repeat this code and I think it's because it was exported from `gateway-bootnode.ts` and then imported from there. It makes much more sense to have it in `common.ts` but this has to be done after the rebase.